.github/workflows/push.yml

```diff
@@ -272,7 +284,7 @@ jobs:
   retag-unaffected:
     runs-on: ubuntu-20.04
     timeout-minutes: 5
-    if: always() && needs.pre-checks.result == 'success' && needs.prepare.result != 'skipped'
+    if: always() && needs.pre-checks.result == 'success' && needs.prepare.result != 'skipped' && !startsWith(github.ref, 'feature/')
```

Comment:
So we are not re-tagging unaffected on feature branches?

Reply:
no - do we need to? we're only deploying the changed images.